packages/graphql/src/examples/starwars/index.ts

```diff
@@ -0,0 +1,3 @@
+export { schema } from './schema';
+export { context } from './simulation/context';
+export { scenarios } from './simulation/scenarios';
```

Comment:
We need to export these from a module since the 'graphql' simulator accepts these options as dynamic imports.

It's a little undesirable to have this example schema in the 'graphql' simulator package itself.

Should we introduce a `@simulacrum/graphql-starwars-schema` package?

Reply:
Maybe just `@simulacrum/graphql-starwars` since the simulator actually will run an entire graphql API. The idea being that if you could just do something like:

```ts
import { createSimulationServer } from '@simulacrum/server';
import { starwars }  from '@simulacrum/graphql-starwars';

export function createServer() {
  return createSimulationServer({ simulators: { starwars } });
}
```